test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.ConnectionLifecycle.cs

```diff
@@ -142,7 +145,13 @@ Task OnTransportStart()
                     {
                         Assert.Equal(0, startCounter);
                         await connection.StartAsync(TransferFormat.Text);
+                        if (!IsWebSocketsSupported())
```

Comment:
Is this a better or worse solution than having separate tests that are or are not conditional on web sockets? Thoughts @anurse @davidfowl ?

Reply:
I think it's fine this way. We don't want piles of skipped tests (they create some noise).

@mikaelm12 can you do this by doing `if(!IsWebSocketsSupported()) { passThreshold -= 1; }` and then just having a single Assert instead though?